src/lint/practice_exercises.nim

```diff
@@ -22,7 +22,7 @@ proc isValidPracticeExerciseConfig(data: JsonNode, path: Path): bool =
                         uniqueValues = true),
       hasArrayOfStrings(data, "contributors", path, isRequired = false,
                         uniqueValues = true),
-      if false: hasValidFiles(data, path) else: true,
+      hasValidFiles(data, path, exerciseDir),
```

Comment:
I enabled this rule again, as we've done a lot of bulk PRs to get these values populated for tracks. At this point, I _want_ configlet to complain about missing files :)